client.js

```diff
@@ -0,0 +1,20 @@
+import {renderElement} from 'nomplate';
+
+import constitutional from './src/constitutional';
+import components from './src/components';
+
+if (require.main === module) {
+  global.window.onload = () => {
+    const element = renderElement(constitutional(), global.document);
```

Comment:
I had a realization that we can pull this line out of the onload handler and do half of our work while the browser is waiting for network shizz.